### PR TITLE
Add edit

### PR DESCRIPTION
**Pull request by oceanelf**

### CODE REVIEW

File: app/controllers/cars_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class CarsController < ApplicationController
+  before_action :find_car, only: [:show, :edit, :update]
```
Comment (JakeM93) [inline app/controllers/cars_controller.rb:2]: Looks good but seems to be conflict from both of our PRs
@oceanelf i can look into this if you want? from what i can see want to keep both bits at the top

**Comment by JakeM93** [comment]
Looks good :) checked out and tried it 🎉 
Sorted some conflicted changes.

File: app/controllers/cars_controller.rb (hunk 1 above)
Comment (JakeM93) [reply app/controllers/cars_controller.rb:3]: Merge conflict sorted

**Review by JakeM93 — APPROVED** [review]
Woop Woop 👍

File: app/controllers/cars_controller.rb (hunk 1 above)
Comment (oceanelf) [reply app/controllers/cars_controller.rb:3]: thank you:))

**Pull request merged**